cli/cli/Services/ProjectContextUtil.cs

```diff
@@ -559,10 +560,7 @@ public static void UpdateUnityProjectReferences(BeamoServiceDefinition definitio
 		const string ITEM_TYPE = "ProjectReference";
 
 		var buildEngine = new ProjectCollection();
-		var stream = File.OpenRead(definition.ProjectPath);

```

Comment:
This wasn't working on nightly builds, this particularly line crashed because couldn't parse parts of the csproj that invoke methods to get the actual version. I'm not sure why this was happening, but changing this fixed that and also improved the formatting that was happening in the project when saving it to the file.